packages/dashboard/src/views/Holdings.js

```diff
@@ -101,16 +101,34 @@ const Holdings = ({
       const currAllData = historicQuotes['all']
       const currAllResults = currAllData.results[currAllData.results.length - 1]
 
-      // override current balance
+      // Override cash value (buying power)
       smallStats[0] = {
         ...smallStats[0],
+        value: Math.round(accounts["non-registered-ijzmubrj"].buying_power.amount * 100) / 100
+      }
+
+      // Override securities value
+
+      const positions = accounts["non-registered-ijzmubrj"].positions
```

Comment:
This isnt gonna work, every persons account is going to be different. Same with line 107. You need to programmatically pull this from `selectedAccount` I think its called.